code/game/atom/atom_act.dm

```diff
@@ -0,0 +1,30 @@
+/*
+ * +++++++++++++++++++++++++++++++++++++++++ ABOUT THIS FILE +++++++++++++++++++++++++++++++++++++++++++++
+ * Not everything here necessarily has the name pattern of [x]_act()
+ * This is a file for various atom procs that simply get called when something is happening to that atom.
+ * If you're adding something here, you likely want a signal and SHOULD_CALL_PARENT(TRUE)
+ * +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+ */
+
+/**
+ * Respond to fire being used on our atom
+ *
+ * Default behaviour is to send [COMSIG_ATOM_FIRE_ACT] and return
+ */
+/atom/proc/fire_act(exposed_temperature, exposed_volume)
```

Comment:
would be nice to comment what `exposed_temperature` and `exposed_volume` are supposed to mean
is temperature in kelvins? volume in...?

Reply:
The variables should be documented.
If someone who is unfamiliar with the system looks at exposed_temperature and exposed_volume they should be able to look at a description to find out what purpose those parameters have and what values they can expect to receive.

It not being documented on TG is not relevant for us.
If it bothers you, you can always PR the description to tg as well.